api/src/plugins/fastify-sentry.ts

```diff
@@ -2,6 +2,7 @@ import Sentry, { SentryPluginOptions } from '@immobiliarelabs/fastify-sentry';
 import { FastifyPluginCallback } from 'fastify';
 import fp from 'fastify-plugin';
 
+/* eslint-disable @typescript-eslint/no-misused-promises */
 const fastifySentry: FastifyPluginCallback<SentryPluginOptions> = async (
```

Comment:
This is a code smell: there shouldn't be a need to disable eslint here, but the type annotation is wrong (it's expecting a callback, not async/await).

I'll take care of that in another PR.

Reply:
tag me when you do, i'd like to see what you come up with up. i fooled around with it for a while and couldn't figure it out.